contract_isp/contract_isp_view.xml

```diff
@@ -1,180 +1,176 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <openerp>
-    <data>
-	<record id="view_company_contract_isp_form" model="ir.ui.view">
-            <field name="name">contract.isp.res.company.form</field>
-            <field name="model">res.company</field>
-	    <field name="inherit_id" ref="base.view_company_form"/>
-            <field name="arch" type="xml">
-		<xpath expr="//page[@string='Configuration']" position="after">
-		    <page string="ISP Contract Management">
-			<group>
-			    <group>
-				<field name="parent_account_id" />
-			    </group>
+        <data>
+            <record id="view_company_contract_isp_form" model="ir.ui.view">
+                <field name="name">contract.isp.res.company.form</field>
+                <field name="model">res.company</field>
+                <field name="inherit_id" ref="base.view_company_form"/>
+                <field name="arch" type="xml">
+                    <xpath expr="//page[@string='Configuration']" position="after">
+                        <page string="ISP Contract Management">
+                            <group>
+                                <group>
+                                    <field name="parent_account_id" />
+                                </group>
+                                <group>
+                                    <field name="cutoff_day" />
+                                </group>
+                            </group>
+                        </page>
+                    </xpath>
+                </field>
+            </record>
 
-			    <group>
-				<field name="cutoff_day" />
-			    </group>
-			</group>
-		    </page>
-		</xpath>
-	    </field>
-	</record>
+            <record id="view_contract_isp_form" model="ir.ui.view">
+                <field name="name">contract.isp.form</field>
+                <field name="model">account.analytic.account</field>
+                <field name="inherit_id" ref="account_analytic_analysis.account_analytic_account_form_form"/>
+                <field name="arch" type="xml">
+                    <xpath expr='//div[@name="project"]' position='inside'>
+                        <field name="use_contract_services" />
+                        <label for="use_contract_services" />
+                    </xpath>
 
-  <record id="view_contract_isp_form" model="ir.ui.view">
-    <field name="name">contract.isp.form</field>
-    <field name="model">account.analytic.account</field>
-    <field name="inherit_id" ref="account_analytic_analysis.account_analytic_account_form_form"/>
-    <field name="arch" type="xml">
-      <xpath expr='//div[@name="project"]' position='inside'>
-        <field name="use_contract_services" />
-        <label for="use_contract_services" />
-      </xpath>
+                    <xpath expr="//separator[@string='Invoicing']" position="before">
```

Comment:
@jbchouinard Did you change anything other than the identation here?

Reply:
@joaoalf just indent. github's diff program is not doing a very good job of detecting it, it's aligning the diffs weirdly so it looks like I changed a lot. you can check the commits separately, the first commit is the only significant code change, the rest is just so the module passes the linter (indentation, version in the manifest, changed how a sql query is built cause it complained about injection attack cause the code used the .format method, even though the parameters to .format are hardcoded strings, not user input)

Reply:
@joaoalf actually, sorry, I did something weird with the version number because I had trouble merging this with another unmerged pull, but that shouldn't be on OCA, I'll fix it now